lib/master.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @file In order to load huge volumes of polygon data into memory without
+ *    breaking Node (due to its low heap-size limits), the package forks a
+ *    worker process per polygon layer/shapefile. This module contains
+ *    functions for initializing them and searching them.
+ */
+
+'use strict';
+
+var childProcess = require( 'child_process' );
+var peliasConfig = require( 'pelias-config' );
+var path = require( 'path' );
+var through = require( 'through2' );
+var logger = require( 'pelias-logger' ).get( 'admin-lookup:master' );
+
+/**
+ * For every desired administrative layer, specifies its name, the partial path
+ * of the corresponding Quattroshapes file, and the properties to extract from
+ * it.
+ */
+var quattroAdminLevels = [
```

Comment:
Yes, there is a default and I know to override it in my own file, but without looking at the code I couldn't tell which settings needed to be overwritten in the config file. To someone that hasn't used the entire import pipeline and is just curious in this portion, it wouldn't be obvious at all.